packages/cli/src/lib/implementation/filter.middleware.unit.test.ts

```diff
@@ -316,8 +314,15 @@ describe('filterMiddleware', () => {
       verbose: true,
     });
 
-    expect(loggerSpy).toHaveBeenCalledWith(
-      expect.stringContaining('removed the following categories'),
+    expect(ui()).toHaveLoggedNthLevel(1, 'info');
+    expect(ui()).toHaveLoggedNthLevel(2, 'info');
+    expect(ui()).toHaveLoggedNthMessage(
+      1,
+      'The --skipPlugins argument removed the following categories: c1, c2.',
+    );
+    expect(ui()).toHaveLoggedNthMessage(
+      2,
+      'The --onlyPlugins argument removed the following categories: c1, c2.',
     );
```

Comment:
Following up on my previous comment, I would merge the `Nth` assertions also:

```suggestion
    expect(ui()).toHaveLoggedNth(
      1,
      'info',
      'The --skipPlugins argument removed the following categories: c1, c2.',
    );
    expect(ui()).toHaveLoggedNth(
      2,
      'info',
      'The --onlyPlugins argument removed the following categories: c1, c2.',
    );
```